README.md

```diff
@@ -182,13 +179,12 @@ The certificate is also printed to the controller log on startup.
 Since v0.9.x certificates get automatically renewed every 30 days. It's good practice that you and your team
 update your offline certificate periodically. To help you with that, since v0.9.2 `kubeseal` accepts URLs too. You can setup your internal automation to publish certificates somewhere you trust.
 
-```bash
```

Comment:
I'm in two minds about this.

I love `console` because it clearly distinguishes the command from it's output. This is useful when rendering examples of interactions.

However using `console` forces you to put a prompt at the start of the line. This makes it harder for people to copy paste the command, since now they have to strip the `$`

Reply:
I totally agree that stripping the `$` is annoying. This is specially painful when you need to copy/paste a script including several lines.
That's why I'm going to revert this approach and use `console` exclusively when we want to show the command's output.